packages/next/build/index.ts

```diff
@@ -75,12 +77,28 @@ export default async function build (dir: string, conf = null): Promise<void> {
       serverlessEntrypoints[errorPage] = `next-serverless-loader?${stringify(serverlessLoaderOptions)}!`
     }
 
-    entrypoints = serverlessEntrypoints
+    serverEntrypoints = serverlessEntrypoints
   }
+  
+  const clientEntrypoints: any = {}
+
+  Object.keys(pages).forEach(async (page) => {
```

Comment:
Is async needed here?
Also, I saw for..in instead of Object.forEach in another parts of the code. Maybe would be better to use for..in if possible to keep consistency.

Reply:
You're looking at outdated code 😄